### DISCUSSION TITLE
skynet coroutine 相关方法讨论

### DISCUSSION THREAD

**Discussion by cloudfreexiao** [cloudwu/skynet#1588] [Q&A]
需要 对接 外部 库时 需要 重写 coroutine 相关方法 但是 skynet 创建 携程是 本地 保存的
是否 能 修改下 不用 本地 保存的 方法
https://github.com/lvzixun/luaprofile/blob/master/profile.lua#L15
https://github.com/cloudwu/skynet/blob/master/lualib/skynet.lua#L252

**Comment by cloudwu** [comment]
你在 require 之前，比如 preload 里就可以改掉。
